docs/app/js/.eslintrc.json

```diff
@@ -0,0 +1,274 @@
+    "no-return-assign": "off",
+    "no-return-await": "error",
+    "no-script-url": "error",
+    "no-self-compare": "error",
+    "no-sequences": "off",
+    "no-shadow": "off",
+    "no-shadow-restricted-names": "error",
+    "no-spaced-func": "off",
+    "no-sync": "error",
+    "no-tabs": "off",
+    "no-template-curly-in-string": "error",
+    "no-ternary": "off",
+    "no-throw-literal": "off",
+    "no-trailing-spaces": "off",
+    "no-undef-init": "error",
+    "no-undefined": "off",
+    "no-underscore-dangle": "off",
+    "no-unmodified-loop-condition": "error",
+    "no-unneeded-ternary": "off",
+    "no-unused-expressions": "off",
+    "no-use-before-define": "off",
+    "no-useless-call": "error",
+    "no-useless-computed-key": "error",
+    "no-useless-concat": "off",
+    "no-useless-constructor": "error",
+    "no-useless-rename": "error",
+    "no-useless-return": "error",
+    "no-var": "off",
+    "no-void": "off",
+    "no-warning-comments": "off",
+    "no-whitespace-before-property": "error",
+    "no-with": "error",
+    "nonblock-statement-body-position": [
+      "error",
+      "any"
+    ],
+    "object-curly-newline": "off",
+    "object-curly-spacing": "off",
+    "object-property-newline": "off",
+    "object-shorthand": "off",
+    "one-var": "off",
+    "one-var-declaration-per-line": "off",
+    "operator-assignment": "off",
+    "operator-linebreak": "off",
+    "padded-blocks": "off",
+    "padding-line-between-statements": "error",
+    "prefer-arrow-callback": "off",
+    "prefer-const": "error",
+    "prefer-destructuring": "off",
+    "prefer-numeric-literals": "error",
+    "prefer-promise-reject-errors": "error",
+    "prefer-reflect": "off",
+    "prefer-rest-params": "off",
+    "prefer-spread": "off",
+    "prefer-template": "off",
+    "quote-props": "off",
+    "quotes": "off",
+    "radix": "off",
+    "require-await": "error",
+    "require-jsdoc": "off",
+    "rest-spread-spacing": "error",
+    "semi": "off",
+    "semi-spacing": "off",
+    "semi-style": "off",
+    "sort-imports": "error",
+    "sort-keys": "off",
+    "sort-vars": "off",
+    "space-before-blocks": "off",
+    "space-before-function-paren": "off",
+    "space-in-parens": "off",
+    "space-infix-ops": "off",
+    "space-unary-ops": "off",
+    "spaced-comment": "off",
+    "strict": "off",
+    "switch-colon-spacing": "off",
+    "symbol-description": "error",
+    "template-curly-spacing": "error",
+    "template-tag-spacing": "error",
+    "unicode-bom": [
+      "error",
+      "never"
+    ],
+    "valid-jsdoc": "off",
+    "vars-on-top": "off",
+    "wrap-iife": "off",
+    "wrap-regex": "off",
+    "yield-star-spacing": "error",
+    "yoda": "off",
+    "no-unused-vars": "off",
+    "no-cond-assign": "off",
+    "no-unexpected-multiline": "off"
+  },
+  "env": {
+    "browser": true
+  },
+  "globals": {
+    "angular": true,
+    "CryptoJS": true,
+    "hljs": true
+  },
+  "overrides": [
```

Comment:
eg this overrides list can add an object with `files: ["docs/"]` configured for 'docs'.

Reply:
I see that `overrides` is an array... oh and it has a seperate `rules`, `globals`, and `env` that applies to that set of `files`. Oh very nice. Then I can do it all in that one file, excellent. I'll work on merging things together. Thank you!